index.js

```diff
@@ -54,11 +56,15 @@ class Analytics {
       enumerable: true,
       value: typeof options.enable === 'boolean' ? options.enable : true
     })
-    axiosRetry(this.axiosInstance, {
-      retries: options.retryCount || 3,
-      retryCondition: this._isErrorRetryable,
-      retryDelay: axiosRetry.exponentialDelay
-    })
+    if (options.retryCount !== 0) {
```

Comment:
here it should be `=== 0`, right?

Reply:
No? If the user sets a `retryCount` specifically of `0` then we do not need `axiosRetry` :)

Reply:
You're right. Sorry

Reply:
No worries :D Its always good to double check :)
Appreciate the review!